docs/cypress/e2e/links.cy.ts

```diff
@@ -0,0 +1,136 @@
+import { VALIDATED_LINKS } from '../data/validatedLinks';
+import { REQUEST_GET_LINKS } from '../data/requestGetLinks';
+import { BASE_URL } from '../data/constants';
+
+let allLinks: string[] = [];
+const numberOfLinks = 119;
+const requestedLinks: Set<string> = new Set();
+
+before(() => {
+  cy.task('readSitemapLinks').then((links: string[]) => {
+    allLinks = allLinks.concat(links);
+  });
+});
+
+describe('Local Sitemap', () => {
+  it('should have 119 links', () => {
+    expect(allLinks.length).to.eq(numberOfLinks);
+  });
+});
+
+describe(`All pages on Sitemap`, () => {
+  for (let i = 0; i < numberOfLinks; i++) {
+    it(`all links on page ${i} should work`, () => {
+      const link = allLinks[i];
+      cy.task('log', `🧪[TESTING...] page ${BASE_URL}/${link}`);
+      cy.visit(link || '/');
+      cy.get('a').each(hrefWorks);
+
+      function hrefWorks(htmlTag: JQuery<HTMLElement>): void {
```

Comment:
`JQuery`? 🤔

Reply:
hahaha, yea, that's what `cy.get()` returns [ref](https://docs.cypress.io/api/commands/get)

![image](https://user-images.githubusercontent.com/11983489/199111129-a1512a00-9a85-4895-8855-6f4fe5a64f3a.png)

Reply:
But do we need to use `JQuery` types?

Reply:
I think so, since we use some `JQuery` methods in line30-33, like `htmlTag.prop('href')`

Reply:
Are those only `JQuery` methods?

Reply:
yes, it's actully from the type of `cy.each`

```ts
each<E extends Node = HTMLElement>(fn: (element: JQuery<E>, index: number, $list: E[]) => void): Chainable<JQuery<E>>
```

Reply:
Got it, thanks